Jenkinsfile.release

```diff
@@ -31,10 +39,13 @@ ansiColor('xterm') {
           sshagent (credentials: ['0f7ec9c9-99b2-4797-9ed5-625572d5931d']) {
             // add the ssh host key for the package uploader step
             // These environment variables are defined in the "Properties Content" section for the job.
-            sh """echo "${PKG_SSH_HOST} ${PKG_SSH_HOSTKEY_TYPE} ${PKG_SSH_HOSTKEY}" | sudo -E tee -a sudo -E tee -a /etc/ssh/ssh_known_hosts"""
+            sh """echo "${PKG_SSH_HOST} ${PKG_SSH_HOSTKEY_TYPE} ${PKG_SSH_HOSTKEY}" | sudo -E tee -a /etc/ssh/ssh_known_hosts"""
 
             withDockerRegistry([credentialsId: 'docker-hub-credentials']) {
-              sh """sudo -E ci/pipeline release $params.version $params.gitsha $params.latest"""
+              sh """if [ \$(md5sum /usr/local/bin/amm | awk '{print \$1}') = '07143cdf89e94b0a957f5656193d88d7' ]; then sudo curl -L -o /usr/local/bin/amm https://github.com/lihaoyi/Ammonite/releases/download/1.0.3/2.12-1.0.3 && sudo chmod +x /usr/local/bin/amm; fi"""
```

Comment:
What is the reason for provisioning Ammonite here?

Reply:
Because the version in the AMI image is insufficient and I cannot update the AMI image right now. Blocked on credentials.

Reply:
https://jira.mesosphere.com/browse/MARATHON-8057